android/modules/ui/src/java/ti/modules/titanium/ui/android/AndroidModule.java

```diff
@@ -99,11 +99,11 @@
 	public static final int SOFT_KEYBOARD_SHOW_ON_FOCUS = TiUIView.SOFT_KEYBOARD_SHOW_ON_FOCUS;
 
 	@Kroll.constant
-	public static final int SWITCH_STYLE_CHECKBOX = 0;
+	public static final int SWITCH_STYLE_CHECKBOX = UIModule.SWITCH_STYLE_CHECKBOX;
 	@Kroll.constant
-	public static final int SWITCH_STYLE_TOGGLEBUTTON = 1;
+	public static final int SWITCH_STYLE_TOGGLEBUTTON = UIModule.SWITCH_STYLE_TOGGLEBUTTON;
```

Comment:
Maybe a good time to rename `SWITCH_STYLE_TOGGLEBUTTON` to `SWITCH_STYLE_TOGGLE_BUTTON`?

Reply:
I agree. I renamed the constant under `Ti.UI`, but kept the old name under `Ti.U.Android`.